perllib/FixMyStreet/DB/Result/Problem.pm

```diff
@@ -941,7 +941,9 @@ sub duration_string {
     my $cobrand = $problem->result_source->schema->cobrand;
     my $body = $cobrand->call_hook( link_to_council_cobrand => $problem )
         || $problem->body(1);
-    return unless $problem->whensent;
+    return
+        unless $problem->whensent
+        && ( $problem->external_id // '' ) !~ /^JOB_/;
```

Comment:
I wonder if this would be better to look at `$problem->service` and skip if that's `Open311`? That way we'd never show the duration string for any fetched reports.

Reply:
I may need to double-check this, but it looks like this removes the 'Sent to...' string for reports which originated on FMS as well.